File: java/teste/src/DesafioRaphael/atvPainelPet/window.java
package DesafioRaphael.atvPainelPet;

import java.io.IOException;
import java.util.Scanner;

public class window {
	public static void main(String[] args) throws InterruptedException, IOException {
		windowIndex();
	}
// ------------------------------------------------------------------------------------------------------
	public static void windowIndex() throws InterruptedException, IOException {
		process.clearTerminal();
		Scanner in = new Scanner(System.in);
		while(true) {
			System.out.println("---------- Welcome PetShop ----------");
			System.out.println("(1) - Cadastrar pessoa");
			System.out.println("(2) - cadastrar pet");
			System.out.println("(3) - Encerrar programa");
			System.out.println("");
			System.out.print("Digite sua escolha: ");
			int escolha = in.nextInt();
			
			if(escolha == 1) {
				windowPerson();
			}else if(escolha == 2){
				windowPet();
			}else{
				process.killProcess();
				break;
			}
			
		}	
	}
// ------------------------------------------------------------------------------------------------------
	public static void windowPerson() throws InterruptedException, IOException {
		process.clearTerminal();
		
		Scanner in = new Scanner(System.in);
		while(true) {
			System.out.println("---------- Cadastro da pessoa ----------");
			System.out.print("Cadastrar nome: ");
			String nome = in.next();
			
			System.out.print("Cadastrar telefone: ");
			String tel = in.next();
			
			System.out.print("Cadastrar endereço: ");
			String address = in.next();
			
			System.out.println("(1) - cadastrar pet");
			System.out.println("(2) - voltar menu");
			System.out.println("(3) - Encerrar programa");
			System.out.println("");
			System.out.print("Digite sua escolha: ");
			int escolha = in.nextInt();
			
			if(escolha == 1) {
				windowPet();
			}else if(escolha == 2){
				windowIndex();
			}else{
				process.killProcess();
				break;
			}
			
		}
	}
// ------------------------------------------------------------------------------------------------------
	public static void windowPet() throws InterruptedException, IOException {
		process.clearTerminal();
		Scanner in = new Scanner(System.in);
		
		while(true) {
			System.out.println("---------- Cadastro do Pet ----------");
			System.out.print("Cadastrar nome pet: ");
			String nome = in.next();
			
			System.out.print("Cadastrar idade pet: ");
			String idade = in.next();
			
			System.out.print("Cadastrar raça pet: ");
			String raca = in.next();
			
			System.out.println("(1) - cadastrar pessoa");
			System.out.println("(2) - voltar menu");
			System.out.println("(3) - Encerrar programa");
			System.out.println("");
			System.out.print("Digite sua escolha: ");
			int escolha = in.nextInt();
			
			if(escolha == 1) {
				windowPerson();
			}else if(escolha == 2){
				windowIndex();
			}else{
				process.killProcess();
				break;
			}
			
		}
	}	
// ------------------------------------------------------------------------------------------------------
}

File: python/prof.Raphael/atvPet/window.py
import process
import os

def windowIndex():
    process.clearTerminal()
    while(True):
        print(5*"-------"," Welcome PetShop ",5*"-------")
        print("(1) - Cadastrar pessoa")
        print("(2) - Cadastrar pet")
        print("(3) - Encerrar programa")
        print("")
        escolha = int(input("Faça sua escolha: "))

        if escolha == 1:
            windowPerson()
        elif escolha == 2:
            windowPet()
        else:
            process.clearTerminal()
            process.stopProcess()


def windowPerson():
    process.clearTerminal()
    while(True):
        print(5*"-------"," Cadastro da pessoa ",5*"-------")
        nome = input("Digite Nome: ")
        telefone = input("Digite Telefone: ")
        endereco = input("Digite Endereço: ")

        print("")
        print("(1) - Cadastrar pet")
        print("(2) - voltar menu")
        print("(3) - Encerrar programa")
        print("")
        escolha = int(input("Faça sua escolha: "))

        if escolha == 1:
            windowPet()
        elif escolha == 2:
            windowIndex()
        else:
            process.clearTerminal()
            process.stopProcess()

def windowPet():
    process.clearTerminal()
    while(True):
        print(5*"-------"," Cadastro do Pet ",5*"-------")
        nome = input("Digite nome Pet: ")
        idade = int(input("Digite idade Pet: "))
        raca = input("Digite raça Pet: ")

        print("")
        print("(1) - Cadastrar pessoa")
        print("(2) - voltar menu")
        print("(3) - Encerrar programa")
        print("")
        escolha = int(input("Faça sua escolha: "))

        if escolha == 1:
            windowPerson()
        elif escolha == 2:
            windowIndex()
        else:
            process.clearTerminal()
            process.stopProcess()

windowIndex()